src/utils/parser/frameworks.js

```diff
@@ -0,0 +1,47 @@
+import getSymbols from "./getSymbols";
+
+export function isReactComponent(source) {
+  const { imports, classes } = getSymbols(source);
+
+  if (!imports || !classes) {
+    return false;
+  }
+
+  if (!importsReact(imports) || !extendsComponent(classes)) {
+    return false;
+  }
+
+  return true;
+}
+
+export function isReduxAction(source) {
+  const { functions } = getSymbols(source);
```

Comment:
lets drop these functions and just do the `isReactComponent` work now

Reply:
Will do 👍